src/sidebar/components/ThreadCard.js

```diff
@@ -63,13 +63,26 @@ function ThreadCard({ frameSync, thread }) {
   // parent component but the `Thread` itself has not changed.
   const threadContent = useMemo(() => <Thread thread={thread} />, [thread]);
 
+  // Handle requests to give this thread keyboard focus.
+  const focusRequest = store.annotationFocusRequest();
+  const cardRef = useRef(/** @type {HTMLElement|null} */ (null));
+  useEffect(() => {
+    if (focusRequest !== thread.id || !cardRef.current) {
+      return;
+    }
+    cardRef.current.focus();
+    store.clearAnnotationFocusRequest();
+  }, [focusRequest, store, thread.id]);
+
   return (
     /* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */
     <Card
       classes={classnames('p-3 cursor-pointer', {
```

Comment:
Suggest adding `focus-visible-ring` to `classes` here.

Without it:

<img width="436" alt="image" src="https://user-images.githubusercontent.com/439947/188638163-1c996f91-6cfa-4b2b-a594-2c36496056da.png">

With it:

<img width="426" alt="image" src="https://user-images.githubusercontent.com/439947/188638274-b2cedeaa-0bc5-48c1-8a50-8f7a11df98dc.png">